CMakeLists.txt

```diff
@@ -75,6 +75,8 @@ add_library(MainLoopLib ${SRC_MAIN} ${MAIN_CONFIG_SRC})
 
 target_include_directories(MainLoopLib PUBLIC ./src/)
 
+target_precompile_headers(MainLoopLib PUBLIC ${PROJECT_SOURCE_DIR}/src/common/pch.h)
+
```

Comment:
Cool! Didn't know this was a thing. Also, where did you learn all those tricks?

Reply:
The first time I've heard about it was in [this video](https://youtu.be/eSI4wctZUto?t=1090), his C++ videos are pretty neat 👌